app/models/server/raw/VoipRooms.ts

```diff
@@ -0,0 +1,171 @@
+import { Cursor, FilterQuery, WithoutProjection, FindOneOptions } from 'mongodb';
+
+import { BaseRaw } from './BaseRaw';
+import { IVoipRoom } from '../../../../definition/IRoom';
+import { Logger } from '../../../../server/lib/logger/Logger';
+
+type T = IVoipRoom;
+export class VoipRoomsRaw extends BaseRaw<T> {
```

Comment:
```suggestion
export class VoipRoomsRaw extends BaseRaw<IVoipRoom> {
```

Reply:
Fixed. Is there any specific reason for not using this shorthand?